public/doc/swagger-2.yaml

```diff
@@ -697,8 +697,16 @@ securityDefinitions:
   approval_auth:
     type: basic
     description: The user id/password is needed when accessing this API externally
```

Comment:
do we support user/password type authentication? if not, let's remove it for now.